examples/shader/post_processing.rs

```diff
@@ -0,0 +1,256 @@
+        reflectance: 0.02,
+        unlit: false,
+        ..default()
+    });
+
+    // The cube that will be rendered to the texture.
+    commands
+        .spawn_bundle(PbrBundle {
+            mesh: cube_handle,
+            material: cube_material_handle,
+            transform: Transform::from_translation(Vec3::new(0.0, 0.0, 1.0)),
+            ..default()
+        })
+        .insert(MainCube);
+
+    // Light
+    // NOTE: Currently lights are ignoring render layers - see https://github.com/bevyengine/bevy/issues/3462
+    commands.spawn_bundle(PointLightBundle {
+        transform: Transform::from_translation(Vec3::new(0.0, 0.0, 10.0)),
+        ..default()
+    });
+
+    // Main camera, first to render
+    commands.spawn_bundle(Camera3dBundle {
+        camera_3d: Camera3d {
+            clear_color: ClearColorConfig::Custom(Color::WHITE),
+        },
+        camera: Camera {
+            target: RenderTarget::Image(image_handle.clone()),
+            ..default()
+        },
+        transform: Transform::from_translation(Vec3::new(0.0, 0.0, 15.0))
+            .looking_at(Vec3::default(), Vec3::Y),
+        ..default()
+    });
+
+    // This specifies the layer used for the post processing camera, which will be attached to the post processing camera and 2d quad.
+    let post_processing_pass_layer = RenderLayers::layer((RenderLayers::TOTAL_LAYERS - 1) as u8);
+
+    let quad_handle = meshes.add(Mesh::from(shape::Quad::new(Vec2::new(
+        size.width as f32,
+        size.height as f32,
+    ))));
+
+    // This material has the texture that has been rendered.
+    let material_handle = post_processing_materials.add(PostProcessingMaterial {
+        source_image: image_handle,
+    });
+
+    // Post processing 2d quad, with material using the render texture done by the main camera, with a custom shader.
+    commands
+        .spawn_bundle(MaterialMesh2dBundle {
+            mesh: quad_handle.into(),
+            material: material_handle,
+            transform: Transform {
+                translation: Vec3::new(0.0, 0.0, 1.5),
+                ..default()
+            },
+            ..default()
+        })
+        .insert(post_processing_pass_layer);
+
+    // The post-processing pass camera.
+    commands
+        .spawn_bundle(Camera2dBundle {
+            camera: Camera {
+                // renders after the first main camera which has default value: 0.
+                priority: 1,
+                ..default()
+            },
+            ..Camera2dBundle::default()
+        })
+        .insert(post_processing_pass_layer);
+}
+
+/// Rotates the cube rendered by the main camera
+fn main_camera_cube_rotator_system(
+    time: Res<Time>,
+    mut query: Query<&mut Transform, With<MainCube>>,
+) {
+    for mut transform in query.iter_mut() {
+        transform.rotation *= Quat::from_rotation_x(1.5 * time.delta_seconds());
+        transform.rotation *= Quat::from_rotation_z(1.3 * time.delta_seconds());
```

Comment:
Slowing down the rotation would probably make this easier to look at. I don’t know if that was the accessibility concern.

Reply:
I agree, IMO the speed is the problem, not the chromatic aberration itself.

Reply:
I slowed it down (blindly via github suggest change) ; I’ll update the preview when I get the chance